test/ha/activator_test.go

```diff
@@ -49,7 +49,7 @@ func TestActivatorHAGraceful(t *testing.T) {
 
 func TestActivatorHANonGraceful(t *testing.T) {
 	// For non-graceful tests, we want the pod to receive a SIGKILL straight away.
-	testActivatorHA(t, ptr.Int64(0), 0.95)
+	testActivatorHA(t, ptr.Int64(0), 0.90)
```

Comment:
It sorta feels like this should be a function of how many activators we have, right?

If we're running 10 and trying to keep ~1 always dead non-gracefully, then I'd expect roughly a 10% reduction in dataplane reliability.  If we run 20 then 5%, if we run 2 then 50%.

We now have this information around L103, so I wonder if we should try that.  WDYT?

Reply:
Would we be running this with a 50% SLO then? I see what you're saying but I feel like that doesn't make a ton of sense honestly. If we'd just run the SLO over the downtimes, then I'd agree but we're running the prober over all the waits etc. as well, where it should succeed just fine :thinking:.